src/platform-apps/channels/message.tsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import classNames from 'classnames';
 import moment from 'moment';
-import { Message as MessageModel } from '../../store/messages';
+import { Message as MessageModel, MediaType } from '../../store/messages';
 import { textToEmojis } from './utils';
 import { LinkPreview } from '../../components/link-preview/';
 import { CloudinaryProvider, provider } from '../../lib/cloudinary/provider';
 interface Properties extends MessageModel {
   className: string;
+  openLightbox: any;
```

Comment:
i don't think the event name should reflect what the consumer is doing in response to the event. .. it should just reflect what has happened from the perspective of this component. as a message, i shouldn't need to know what my consumer plans to do when i am clicked. i should just tell the consumer that i was clicked.
something like:
```
onImageClick: () => void;
```